digdag-docs/src/command_reference.rst

```diff
@@ -356,6 +356,7 @@ In the config file, following parameters are available
 * database.idleTimeout (seconds in integer, default: 600)
 * database.validationTimeout (seconds in integer, default: 5)
 * database.maximumPoolSize (integer, default: available CPU cores * 32)
+* database.leakDetectionThreshold (HikariCP leakDetectionThreshold in integer. default: 0. To enable, set to >= 2000.)
```

Comment:
Could you add unit description(`milliseconds`)?

Reply:
@hiroyuki-sato Thanks, I will fix it later.